CHANGELOG.md

```diff
@@ -1,10 +1,24 @@
 # Changelog
 
 ## 1.6.1 -- Unreleased
+
+### Deprecation Warnings
+* `AwsCrypto.encryptString()` and `AwsCrypto.decryptString()` are now deprecated.
+  Please move to `AwsCrypto.encryptData()` and `AwsCrypto.decryptData()` with manual Base64 encoding/decoding.
```

Comment:
```suggestion
Replace your calls to these methods with calls to  `AwsCrypto.encryptData()` and `AwsCrypto.decryptData()`. These methods use manual Base64 encoding/decoding, so they are fully compatible with other language implementations of the AWS Encryption SDK.
```

Reply:
The new methods don't use Base64 encoding and will require the developer to apply it themselves. Do you have better wording for that?

Reply:
Replace your calls to these methods with calls to  `AwsCrypto.encryptData()` and `AwsCrypto.decryptData()`. Because these methods don't perform any Base64 encoding or decoding, they are fully compatible with other language implementations of the AWS Encryption SDK.

Reply:
Replace your calls to these methods with calls to AwsCrypto.encryptData() and AwsCrypto.decryptData(). Unlike the deprecated methods, these methods don't perform any Base64 encoding or decoding, so they are fully compatible with other language implementations of the AWS Encryption SDK.

If you need Base64 encoding or decoding for your application, you can add it outside of the AWS Encryption SDK.